src/pylyrics/download_data.py

```diff
@@ -29,6 +29,15 @@ def download_data(dataset, file_path, columns):
     download_data("geomack/spotifyclassification", "data/spotify_attributes", ("song_title", "artist"))
     spotify_df = download_data("geomack/spotifyclassification", "data/spotify_attributes", ("song_title", "artist"))
     """
+    if not (type(dataset)) == str:
```

Comment:
try should be on top of exception handling

Reply:
Updated the exception handling as suggested and it works as expected.